tests/Console/ModelsCommand/migrations/____simple_table.php

```diff
@@ -8,7 +8,7 @@ class SimpleTable extends Migration
 {
     public function up(): void
     {
-        Schema::create('simple', function (Blueprint $table) {
+        Schema::create('simples', function (Blueprint $table) {
```

Comment:
This is the typo fix, the rest is just adaptions of the tests which now properly "see" the database table